scipy/optimize/_nnls.py

```diff
@@ -5,7 +5,7 @@
 __all__ = ['nnls']
 
 
-def nnls(A, b, maxiter=None, *, atol=None):
+def nnls(A, b, maxiter=None, atol=None):
```

Comment:
```suggestion
def nnls(A, b, maxiter=None, *, atol=_NoValue):
```
Please also add `from script._lib.deprecation import _NoValue` to the imports